src/Emitter/SapiEmitter.php

```diff
@@ -10,6 +10,12 @@
 final class SapiEmitter implements EmitterInterface
 {
     private const NO_BODY_RESPONSE_CODES = [204, 205, 304];
+    private $partSize;
+
+    public function __construct(int $partSize = 8388608)
```

Comment:
What is 8388608? Worth moving to a constant such as `private DEFAULT_BUFFER_SIZE` and adding phpdoc saying why it is exactly 8 megabytes.